DOCS/man/mpv.rst

```diff
@@ -1246,7 +1246,10 @@ PROTOCOLS
     ``<number>`` (select playlist with the same index). mpv will list
     the available playlists on loading.
 
-    ``bluray://`` is an alias.
+.. warning::
+
+    Since mpv 0.39.0, the old alias ``bluray://`` was removed and will now
+    instead map to the ffmpeg protocol with the same name if it is available.
```

Comment:
Maybe?
```suggestion
    Since mpv 0.39.0, the old alias ``bluray://`` has been removed and will now
    map to the ffmpeg protocol with the same name, if it is available.
```

Reply:
Changed it to has been but no comma.